### ISSUE TITLE
Add Babel-Polyfill to preview config

### ISSUE THREAD

**Issue by thomaslloyd** [sky-uk/blackjack-toolbelt#42]
Currently the preview command does not use babel-polyfill, so some things are breaking. E.g., using `Map` breaks the preview in IE9/10.

I've tried adding it to the Webpack entry  point config as suggested [here][0] but this just throws other errors.

[0]: https://babeljs.io/docs/usage/polyfill/

**Comment by joshnesbitt** [comment]
I've started to add this in #43 but there's an odd error that you highlighted regarding IE. I don't actually think the issue is with the polyfill itself but more how we're requiring. Will keep digging.

**Comment by romidane** [comment]
Perhaps the need to add polyfills should be up to the developer?

**Comment by thomaslloyd** [comment]
@romidane Yeah I'd be happy with that. I should've emphasised that the main issue is that no matter where `babel-polyfill` is imported, it throws an error on IE9/10/11 causing the component not to load. I've had a look with @joshnesbitt and recreated it on a freshly `init`ed component, simply by importing it in `preview.js` and running the preview command. We reckon there's something not quite right with the webpack config.

**Comment by romidane** [comment]
@thomaslloyd I've had issues in the past with the babel polyfill. It's doesn't play well with webpack unless it's imported inside the main file of your component in this case the preview file. Try importing it at the top of your preview file ```import 'babel-polyfill'; ```

**Comment by joshnesbitt** [comment]
@romidane yeah this is where we're seeing the issue. Even if the toolbelt ignores all polyfill bits and leaves that to the project to implement, we're seeing the errors.

**Comment by joshnesbitt** [comment]
Fixed in #57.
